lib/symengine/complex.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class Complex
+    def to_c
+        self.to_s.sub('I', 'i').sub('*','').gsub(' ','').to_c
```

Comment:
* unnecessary self;
* `tr` is better than `sub` in this case: `tr('I', 'i').tr('* ', '')`